blue_bringup/scripts/gripper_calibration_service.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python
+
+"""This node provides a service to calibrate the grippers."""
+
+import rospy
+import numpy as np
+import actionlib
+from blue_msgs.srv import GripperPositionCalibration
+from controller_manager_msgs.srv import SwitchController
+from controller_manager_msgs.srv import LoadController
+from controller_manager_msgs.srv import ListControllers
+from std_srvs.srv import Trigger, TriggerRequest, TriggerResponse
+from blue_msgs.msg import MotorState
+from std_msgs.msg import Float64MultiArray
+from control_msgs.msg import (
+    GripperCommandAction,
+    GripperCommandGoal,
+)
```

Comment:
extra nitpicky but the style here isn't consistent with the way you imported multiple things from std_srvs.srv